cluster/gce/util.sh

```diff
@@ -746,6 +746,7 @@ function construct-linux-kubelet-flags {
       #TODO(mikedanese): allow static pods to start before creating a client
       #flags+=" --bootstrap-kubeconfig=/var/lib/kubelet/bootstrap-kubeconfig"
       #flags+=" --kubeconfig=/var/lib/kubelet/kubeconfig"
+      flags+=" --register-with-taints=node-role.kubernetes.io/master=:NoSchedule"
```

Comment:
I think it should just be `flags+=" --register-with-taints=node-role.kubernetes.io/master:NoSchedule"` (omit the final equals sign, just `:` instead of `=:`).

Reply:
Oh nevermind, you have an empty value. That's confusing lol.